Quintet-iOS/View/ElementView.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  ElementView.swift
+//  Quintet-iOS
+//
+//  Created by 옥재은 on 2023/11/10.
+//
+
+import SwiftUI
+
+//요소별 위에 5개 카드
+struct RecordElementView : View {
+    @State var recordIndex: recordElement
+    @State private var isShowPopup = false
+    @StateObject private var viewModel = DateViewModel()
+    @ObservedObject private var coreDataViewModel = CoreDataViewModel()
+    
+    var healthRecords: [RecordMetaData] {
+        return coreDataViewModel.getHealthRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var workRecords: [RecordMetaData] {
+        return coreDataViewModel.getWorkRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var relationshipRecords: [RecordMetaData] {
+        return coreDataViewModel.getRelationshipRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var assetRecords: [RecordMetaData] {
+        return coreDataViewModel.getAssetRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var familyRecords: [RecordMetaData] {
+        return coreDataViewModel.getFamilyRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    
+    func displayRecords(for recordElement: recordElement, records: [RecordMetaData]) -> some View {
+        VStack {
+            Divider()
+                .padding(.horizontal)
+            Button(action: {
+                isShowPopup = true
+            }) {
+                HStack {
+                    viewModel.yearMonthButtonTextRecordVer
+                        .foregroundColor(Color.black)
+                    Image(systemName: isShowPopup ? "chevron.up" : "chevron.down")
+                        .foregroundColor(.black)
+                        .font(.system(size: 15))
+                }
+                .padding(.top, 25)
+                .padding(.trailing, 180)
+            }
+            .sheet(isPresented: $isShowPopup) {
+                CalendarYearMonthPickerPopup(viewModel: viewModel, isShowPopup: $isShowPopup)
+                    .frame(width: 300, height: 400)
+                    .background(BackgroundClearView())
+                    .ignoresSafeArea()
+            }
+            ForEach(records) { metaData in
+                ForEach(metaData.records) { record in
+                    recordCard(icon: record.icon, title: record.title, subtitle: record.subtitle)
```

Comment:
```suggestion
                    recordCard(
                        icon: record.icon,
                        title: record.title,
                        subtitle: record.subtitle
                    )
```
위와 같이 작성하는건 어떨까요? 좀 더 한눈에 알아볼 수 있을것 같습니다!